src/test/java/fr/insalyon/citi/golo/compiler/CompileAndRunTest.java

```diff
@@ -1353,6 +1353,53 @@ public void adapters() throws Throwable {
   }
 
   @Test
+  @SuppressWarnings("unchecked")
+  public void adaptersHelper() throws Throwable {
+
+    try {
+
+      Class<?> moduleClass = compileAndLoadGoloModule(SRC, "adapters-helper.golo");
+
+      Method serializable = moduleClass.getMethod("serializable");
+      Object result = serializable.invoke(null);
+      assertThat(result, notNullValue());
+      assertThat(result, instanceOf(Serializable.class));
+
+      Method runnable = moduleClass.getMethod("runnable");
+      result = runnable.invoke(null);
+      assertThat(result, notNullValue());
+      assertThat(result, instanceOf(Object[].class));
+      Object[] array = (Object[]) result;
+      assertThat(array, both(arrayWithSize(3)).and(arrayContaining((Object) 11, (Object) 12, (Object) 13)));
+
+      Method override_toString = moduleClass.getMethod("override_toString");
+      result = override_toString.invoke(null);
+      assertThat(result, notNullValue());
+      String str = result.toString();
+      assertThat(str, both(startsWith(">>>")).and(containsString("@")));
+
+      Method construct_arraylist = moduleClass.getMethod("construct_arraylist");
+      result = construct_arraylist.invoke(null);
+      assertThat(result, notNullValue());
+      assertThat(result, instanceOf(ArrayList.class));
+      ArrayList<String> arrayList = (ArrayList<String>) result;
+      assertThat(arrayList.size(), is(3));
+      assertThat(arrayList, contains("foo", "bar", "baz"));
+
+      if (!bootstraping()) {
+        Method add_arraylist = moduleClass.getMethod("add_arraylist");
+        result = add_arraylist.invoke(null);
+        assertThat(result, instanceOf(List.class));
+        List<String> strList = (List<String>) result;
+        assertThat(strList.size(), is(3));
+        assertThat(strList, contains("foo", "bar", "baz"));
+      }
+    } catch (Throwable t) {
```

Comment:
Remove the `catch` block, otherwise you print the error stack trace, and the test just passes :smiley: